.gitignore

```diff
@@ -20,3 +20,5 @@
 .ruby-gemset
 
 /spec/examples.txt
+
+*.swp
```

Comment:
I think that since this is specific to us vim users rather than applicable to all developers for this repo, it's best to keep this out of each repo's .gitignore and instead set up your global config so that all of these files are ignored from all of your repos. See https://stackoverflow.com/questions/4824188/git-ignore-vim-temporary-files